vital/algo/video_input.cxx

```diff
@@ -56,13 +75,18 @@ ::get_implementation_capabilities() const
 // ------------------------------------------------------------------
```

Comment:
We don't need to do this now, but another thing we should look into eventually is finding all of these and normalizing them to 79 characters line length. (This one is too short.)

Reply:
I fixed these in this file only.  I'll have a script standardize all the files where necessary.